sickbeard/webapi.py

```diff
@@ -2237,6 +2240,22 @@ def run(self):
                 return _responds(RESULT_FAILURE, msg="Status prohibited")
             newStatus = self.status
 
+        # Assume default future status as wanted.
+        default_ep_status = WANTED
+        if self.future_status:
+            # convert the string status to a int
+            for status in statusStrings.statusStrings:
+                if statusStrings[status].lower() == str(self.future_status).lower():
+                    self.future_status = status
+                    break
+            # TODO: check if obsolete
+            if not self.future_status in statusStrings.statusStrings:
+                raise ApiError("Invalid Status")
+            # only allow the status options we want
+            if int(self.future_status) not in (3, 5, 6, 7):
```

Comment:
Please use the variables for status definition here, in case they ever change:

`not in (WANTED, SKIPPED, ARCHIVED, IGNORED):`